test/e2e/govmomi_test.go

```diff
@@ -92,3 +94,26 @@ func initVSphereSession() {
 func terminateVSphereSession() {
 	Expect(vsphereClient.Logout(ctx)).To(Succeed())
 }
+
+func getClusterctlConfigVariable(clusterctlConfigPath, configVariable string) (string, error) {
```

Comment:
Also seems like something that we should make a util exported by CAPI (in a follow-up PR)